python/tests/conftest.py

```diff
@@ -26,6 +32,8 @@
 from rikai.spark.sql import init
 from rikai.spark.utils import init_spark_session
 
+random.seed(time.time())
```

Comment:
Isn't the default seed already random?

Reply:
Ah, you are right, Thanks @chunyang !

Reply:
Oh I meant I don't think you have to call `random.seed()` at all lol